documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/migration/ServletToHandlerDocs.java

```diff
@@ -0,0 +1,643 @@
+            //   - servletRequest.getDateHeader(name)
+            //   - servletRequest.getIntHeader(name)
+            HttpFields requestHeaders = request.getHeaders();
+
+            // Gets the request Content-Type.
+            // Replaces:
+            //   - servletRequest.getContentType()
+            String contentType = request.getHeaders().get(HttpHeader.CONTENT_TYPE);
+
+            // Gets the request Content-Length.
+            // Replaces:
+            //   - servletRequest.getContentLength()
+            //   - servletRequest.getContentLengthLong()
+            long contentLength = request.getHeaders().getLongField(HttpHeader.CONTENT_LENGTH);
+
+            // Gets the request locales.
+            // Replaces:
+            //   - servletRequest.getLocale()
+            //   - servletRequest.getLocales()
+            List<Locale> locales = Request.getLocales(request);
+
+            // Gets the request scheme.
+            // Replaces:
+            //   - servletRequest.getScheme()
+            String scheme = request.getHttpURI().getScheme();
+
+            // Gets the server name.
+            // Replaces:
+            //   - servletRequest.getServerName()
+            String serverName = Request.getServerName(request);
+
+            // Gets the server port.
+            // Replaces:
+            //   - servletRequest.getServerPort()
+            int serverPort = Request.getServerPort(request);
+
+            // Gets the remote host/address.
+            // Replaces:
+            //   - servletRequest.getRemoteAddr()
+            //   - servletRequest.getRemoteHost()
+            String remoteAddress = Request.getRemoteAddr(request);
+
+            // Gets the remote port.
+            // Replaces:
+            //   - servletRequest.getRemotePort()
+            int remotePort = Request.getRemotePort(request);
+
+            // Gets the local host/address.
+            // Replaces:
+            //   - servletRequest.getLocalAddr()
+            //   - servletRequest.getLocalHost()
+            String localAddress = Request.getLocalAddr(request);
+
+            // Gets the local port.
+            // Replaces:
+            //   - servletRequest.getLocalPort()
+            int localPort = Request.getLocalPort(request);
+
+            // Gets the request attributes.
+            // Replaces:
+            //   - servletRequest.getAttributeNames()
+            //   - servletRequest.getAttribute(name)
+            //   - servletRequest.setAttribute(name, value)
+            //   - servletRequest.removeAttribute(name)
+            String name = "name";
+            Object value = "value";
+            Set<String> names = request.getAttributeNameSet();
+            Object attribute = request.getAttribute(name);
+            request.setAttribute(name, value);
+            request.removeAttribute(name);
+            request.clearAttributes();
+            Map<String, Object> map = request.asAttributeMap();
```

Comment:
It would be good to have an example to show that we now return old values. Plus I'm not sure we need to list the APIs that are the same
```suggestion
            // Gets the request attributes.
            // Replaces:
            //   - Object old = servletRequest.getAttribute("name")
            //   - servletRequest.setAttribute("name", "value")
            Object old = request.setAttribute("name", "value");
```